helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/AvscGenerationConfig.java

```diff
@@ -118,6 +134,27 @@ public class AvscGenerationConfig {
      * do so gradually and without compatibility breaks or global coordinated deployments.
      */
     private final boolean addAvro702Aliases;
+    /***
+     * true copies default values provided in the schema
+     */
+    public final boolean retainDefaults;
+    /***
+     * true copies docs provided in the schema
+     */
+    public final boolean retainDocs;
+    /***
+     * true copies alias values provided in for all the fields in the schema
+     */
+    public final boolean retainFieldAliases;
+    /***
+     * true copies All extra json properties for which AvscWriterPlugins were not provided.
```

Comment:
maybe "which are not part of the avro specification and also were not claimed by any plugin" ? some props are wither in the spec or semi-official ("avro.java.string" ?)